src/elements/accordion/src/index.tsx

```diff
@@ -103,7 +105,17 @@ const Accordion: React.FC<Props> & {
         px={{
           color: 'textColor'
         }}
-        onClick={() => setIsOpen(!isOpen)}
+        onClick={() => {
+          if (
+            isOpen &&
+            variant === 'reverse' &&
```

Comment:
Is there a reason this feature is restricted to the `reverse` variant?

Reply:
I thought because with the other accordions you have to close them from the top so even when the content is long you'd already have scrolled back to the right place and wouldn't want to jump around